curation-migration-datasync/helpers/timeConverter.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { getLocalTimeFromScheduledDate } from './timeConverter';
+import { ScheduledSurfaceGuid } from '../dynamodb/types';
+
+describe('time converter', function () {
+  //for timestamp conversion: https://www.epochconverter.com/
+  const testDate = '2022-06-29';
+  const estTimestamp = 1656486000; //add 7 hrs to the given date
+  const berlinTimestamp = 1656464400; //add one hour to the given date
+  const istTimestamp = 1656451800; //subtract one day and set to 930 pm
+  const gmtTimestamp = 1656471600; //add 3 hours with given date
```

Comment:
Nitpick: all three comments that start with "add X hours" should use "to the given date".